src/main/kotlin/com/cognifide/gradle/aem/environment/docker/ContainerHost.kt

```diff
@@ -0,0 +1,63 @@
+package com.cognifide.gradle.aem.environment.docker
+
+import com.cognifide.gradle.aem.common.file.resolver.FileResolver
+import org.gradle.util.GFileUtils
+import java.io.File
+
+/**
+ * File manager for host OS files related specific Docker container.
+ * Provides DSL for e.g creating directories for volumes and providing extra files shared via volumes.
+ */
+class ContainerHost(val container: Container) {
+
+    private val aem = container.aem
+
+    private val logger = aem.logger
+
+    private val docker = container.docker
+
+    val rootDir = File(docker.environment.rootDir, container.name)
+
+    var fileDir = File(rootDir, aem.props.string("environment.container.host.fileDir") ?: "files")
+
+    private val fileResolver = FileResolver(container.aem, fileDir)
+
+    /**
+     * Get file under environment root directory
+     */
+    fun file(path: String) = File(rootDir, path)
+
+    fun files(options: FileResolver.() -> Unit) {
+        fileResolver.apply(options)
+    }
+
+    fun resolveFiles(): List<File> {
+        logger.info("Resolving files for container '${container.name}'")
+        val files = fileResolver.allFiles
+        logger.info("Resolved files for container '${container.name}':\n${files.joinToString("\n")}")
+
+        return files
+    }
+
+    fun resolveFiles(options: FileResolver.() -> Unit): List<File> {
```

Comment:
this provides files on call time / not using `fileResolver` tied to `environmentResolve` task

yes it is not needed anymore / usable optionally but decided to keep as future proof